PokeAlarm/Cache/Cache.py

```diff
@@ -77,6 +80,24 @@ def gym_image(self, gym_id, gym_image=Unknown.REGULAR):
             self._gym_image[gym_id] = gym_image
         return self._gym_image.get(gym_id, get_image_url('icons/gym_0.png'))
 
+    def weather_id(self, s2_cell_id, weather_id=Unknown.REGULAR):
```

Comment:
Perhaps rename these functions `s2_weather_id` or `cell_weather_id` so it more clearly indicates these aren't cached from other events.